tests/database/odbc-with-cksp/enclave-config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "args": [
+        "odbc_app",
+        "classic",
+        ""
```

Comment:
?

Reply:
there are two args for running the application, the 2nd one by default is set to empty value